src/react-app-env.d.ts

```diff
@@ -1 +1,2 @@
 /// <reference types="react-scripts" />
+// declare module 'react-router-dom';
```

Comment:
?

Reply:
Yes this line can be removed becasuse I installed "@types/..." dependencies